text/src/autogluon/text/automm/optimization/lit_module.py

```diff
@@ -174,8 +177,16 @@ def training_step(self, batch, batch_idx):
         -------
         Average loss of the mini-batch data.
         """
-        output, loss = self._shared_step(batch)
-        self.log("train_loss", loss)
+        if self.mixup_alpha > 0:
+            image = batch['timm_image_image']
+            label = batch[self.model.label_key]
+            mix_images, target_a, target_b, lam = mixup(image, label, alpha=self.mixup_alpha)
+            batch['timm_image_image'] = mix_images
+            output, loss = self._shared_step(batch)
+            self.log("train_loss", loss)
+        else:
+            output, loss = self._shared_step(batch)
+            self.log("train_loss", loss)
```

Comment:
We need to have something like this: https://github.com/rwightman/pytorch-image-models/blob/master/train.py#L696-L697 Always have good API design in our mind when adding new stuff.